src/cli/project/export/conda.rs

```diff
@@ -0,0 +1,198 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+
+use itertools::Itertools;
+use miette::IntoDiagnostic;
+use rattler_conda_types::{MatchSpec, Platform};
+
+use crate::cli::LockFileUsageArgs;
+use crate::lock_file::UpdateLockFileOptions;
+use crate::utils::conda_environment_file::{CondaEnvDep, CondaEnvFile};
+use crate::{project, HasFeatures, Project};
+
+// enum to select version spec formatting
+#[derive(clap::ValueEnum, Clone, Debug)]
+pub enum VersionSpec {
+    Manifest,
+    Locked,
+    None,
+}
+
+/// Exports a projects dependencies as an environment.yml
+///
+/// The environment is printed to standard out
+#[derive(Debug, Parser)]
+#[clap(arg_required_else_help = false)]
+pub struct Args {
+    /// The platform to list packages for. Defaults to the current platform.
+    #[arg(long)]
+    pub platform: Option<Platform>,
+
+    /// The path to 'pixi.toml' or 'pyproject.toml'
+    #[arg(long)]
+    pub manifest_path: Option<PathBuf>,
+
+    /// The environment to list packages for. Defaults to the default environment.
+    #[arg(short, long)]
+    pub environment: Option<String>,
+
+    /// Name for environment
+    #[arg(short, long)]
+    pub name: Option<String>,
+
+    /// Dependency spec output method
+    #[arg(long, default_value = "manifest", value_enum)]
+    pub version_spec: VersionSpec,
```

Comment:
```suggestion
    #[arg(long, default="false")]
    pub lock: bool,
```
What do you think about this? to keep it simple.

Reply:
I explained more of my thinking in https://github.com/prefix-dev/pixi/pull/1427/files#r1616283963 but I think there are really 3 different ways that versions are specified in `environment.yml`s, so the arg needs to be able to handle all three. 

How about shortening `version_spec` to `versions`?`pixi project export conda --versions locked` I think that may convey the intent better than `version_spec` to more users.